app/src/main/res/layout/activity_main.xml

```diff
@@ -8,127 +8,150 @@
 
     <TextView
         android:id="@+id/screen_text_example"
-        style="@style/calculatorScreen"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
-    
-    <GridLayout
+        style="@style/CalculatorScreenInterface"
+        android:layout_width="match_parent" />
+
+    <LinearLayout
+        style="@style/linearWeightValue"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:columnCount="@integer/grid_layout_column_count"
-        android:rowCount="@integer/grid_layout_row_count">
+        android:layout_height="match_parent"
+        android:orientation="horizontal">
 
         <Button
             android:id="@+id/button_number_seven"
-            style="@style/calculatorButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            style="@style/CalculatorNumbersButtons"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:text="@string/main_screen_btn_text_seven" />
 
         <Button
             android:id="@+id/button_number_eight"
-            style="@style/calculatorButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            style="@style/CalculatorNumbersButtons"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:text="@string/main_screen_btn_text_eight" />
 
         <Button
             android:id="@+id/button_number_nine"
-            style="@style/calculatorButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            style="@style/CalculatorNumbersButtons"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:text="@string/main_screen_btn_text_nine" />
 
         <Button
             android:id="@+id/button_function_sum"
-            style="@style/calculatorButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            style="@style/CalculatorOperationsButtons"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="1"
```

Comment:
do you need this atribute?

Reply:
Done. Thx

Reply:
Mati try to explain the response of the question to the other collaborator, not put only "done"